src/infinistore.cpp

```diff
@@ -92,6 +91,7 @@ struct Client {
     struct ibv_cq *cq_ = NULL;
     struct ibv_qp *qp_ = NULL;
     bool rdma_connected_ = false;
+    struct ibv_comp_channel *comp_channel = NULL;
```

Comment:
1. ~Client() should free comp_channel.
2. if this is per client, we should call uv_poll_stop to stop before deleting comp_channel.   
3. can you help to add corresponding unit test for        multiple clients(RDMA+local gpu)

Reply:
done, pls check